hardware.inc

```diff
@@ -46,19 +47,16 @@ MACRO rev_Check_hardware_inc
     ENDC
 ENDM
 
-DEF _VRAM        EQU $8000 ; $8000->$9FFF
-DEF _VRAM8000    EQU _VRAM
-DEF _VRAM8800    EQU _VRAM+$800
-DEF _VRAM9000    EQU _VRAM+$1000
-DEF _SCRN0       EQU $9800 ; $9800->$9BFF
-DEF _SCRN1       EQU $9C00 ; $9C00->$9FFF
-DEF _SRAM        EQU $A000 ; $A000->$BFFF
-DEF _RAM         EQU $C000 ; $C000->$CFFF / $C000->$DFFF
-DEF _RAMBANK     EQU $D000 ; $D000->$DFFF
-DEF _OAMRAM      EQU $FE00 ; $FE00->$FE9F
-DEF _IO          EQU $FF00 ; $FF00->$FF7F,$FFFF
-DEF _AUD3WAVERAM EQU $FF30 ; $FF30->$FF3F
-DEF _HRAM        EQU $FF80 ; $FF80->$FFFE
+DEF _VRAM         EQU $8000 ; $8000->$9FFF

```

Comment:
The extra spacing is a bit unnecessary, and muddies the diff a bit. It'd be better if it was reverted.